content/topics/sharing-code-workflows.md

```diff
@@ -0,0 +1,84 @@
+### Enhance research efficiency and compliance
+
+* **Operational Efficiency** – Modular, well-documented software facilitates smoother research processes, from rerunning experiments to adapting methodologies for new data. 
+* **Legal and Ethical Compliance** – Publishing software and data independently of each other as well as associated research articles simplifies compliance with licensing requirements. Adherence with open code mandates set by journals and funders enhances research credibility and broadens dissemination.
+* **Recognition** – Open code obtains scholarly recognition through citations, emphasising the importance of software as a research output.
+
+### Foster collaboration and global innovation
+* **Global Access and Education** – Sharing tools broadens their reach, enabling equitable access and serving as vital resources for training the next generation of scientists, particularly in resource-limited settings.
+* **Community Contributions and Cross-Field Innovation** – Open software encourages community input and interdisciplinary exchanges, leading to more robust, flexible solutions and sparking innovation across research domains.
+
+<a class="link-teal" href="https://www.software.ac.uk/blog/why-should-you-care-about-reproducible-code-and-how-get-started" target="_blank"> <b>Learn more about practical benefits and how to get started <i class="bi bi-box-arrow-up-right"></i></b></a>
+
+<a class="link-teal" href="https://urn.kb.se/resolve?urn=urn:nbn:se:kb:publ-738" target="_blank"><b>Download and read the National Library of Sweden’s national guidelines <i class="bi bi-box-arrow-up-right"></i></b></a>
+
+## How to share code & workflows?
+
+How you share code and workflows depends on your goals—whether you seek transparency, collaboration, or community contributions. Keep in mind that your objectives may evolve over time, and every step towards FAIR practices adds value.  
+
+<a class="link-teal" href="https://github.com/ScilifelabDataCentre/open-science-checklists/blob/main/open_software_checklist.md" target="_blank"> <b>Consult the SciLifeLab Open Science Software Checklist for comprehensive guidance<i class="bi bi-box-arrow-up-right"></i></b></a>
+
+### Where can code and workflows be shared?
+
+To ensure transparency, make a static copy of your code or workflow available for long-term access, independent of any research paper. This should be accompanied by a README file with a clear description, usage instructions, and license(s). Collaborative version control platforms, such as GitHub, are ideal for both sharing and ongoing development. Use tags or releases to reference specific versions of your code.
+
+To enhance FAIRness, consider obtaining a persistent identifier by publishing your code or workflow in a research output repository, such as [Zenodo](https://zenodo.org/) or the [SciLifeLab Data Repository](https://figshare.scilifelab.se/). This can be achieved by linking your GitHub repository or uploading files directly.
```

Comment:
you can get an identifier automatically for different releases by linking GitHub to Zenodo. If you want a general identifier, the RRID. If your code is going to be FAIR, then you could also consider https://fairsharing.org